invoicing/tests/factories.py

```diff
@@ -7,7 +7,10 @@
 
 
 class InstallmentBaseFactory(factory.django.DjangoModelFactory):
-    type = factory.Faker("random_element", elements=list(InstallmentType))
+    # Faker("random_element") cannot be used because it could generate non-unique values
```

Comment:
I prefer to add these kinds of comments to the git commit description instead to code, but this is ok.